perception/map_based_prediction/include/map_based_prediction/path_generator.hpp

```diff
@@ -49,7 +49,31 @@ struct FrenetPoint
   float d_acc;
 };
 
-using EntryPoint = std::pair<Eigen::Vector2d /*in*/, Eigen::Vector2d /*out*/>;
+struct CrosswalkEdgePoints
+{
+  Eigen::Vector2d front_center_point;
+  Eigen::Vector2d front_right_point;
+  Eigen::Vector2d front_left_point;
+  Eigen::Vector2d back_center_point;
+  Eigen::Vector2d back_right_point;
+  Eigen::Vector2d back_left_point;
+
```

Comment:
Front and back means, closer and further edge from pedestrians, right?
I think it is little bit confusing, so please take care when add README documents.